### DISCUSSION TITLE
Error blueman

### DISCUSSION THREAD

**Discussion by simone19959595** [blueman-project/blueman#2565] [Q&A]
ALthoug I tried to follow thousands of blogs, I am able to start the bluetooth services but not to connect to my bluetooth device. See screenshot. It does not even start any scan. See attached screenshot. Any help on how to solve this would be appreciated!

Thanks in advance,

blueman:2.4.3-1
BlueZ: 5.79-1
Distribution: Kali linux 64 bit 2024.4
Desktop environment: xfce 

* [ ] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

![Screenshot_2024-12-15_17-21-16](https://github.com/user-attachments/assets/e59d10b1-3fbe-4377-b3fa-04e12e256077)

**Comment by cschramm** [comment]
It looks like your system simply does not provide any adapter.

**Reply by simone19959595** [reply]
OK, so how can I add an adapter to my system? I've been googling around a lot but I cannot find any reasonable solution
